share/ramble/ramble-completion.bash

```diff
@@ -267,7 +267,16 @@ _ramble() {
     then
         RAMBLE_COMPREPLY="-h --help -H --all-help --color -c --config -C --config-scope -d --debug --timestamp --pdb -w --workspace -D --workspace-dir -W --no-workspace --use-workspace-repo -k --insecure -l --enable-locks -L --disable-locks -m --mock -p --profile --sorted-profile --lines -v --verbose --stacktrace -V --version --print-shell-vars"
     else
-        RAMBLE_COMPREPLY="clean commands config debug edit flake8 help info license list mirror mods on repo software-definitions unit-test workspace"
+        RAMBLE_COMPREPLY="attributes clean commands config debug edit flake8 help info license list mirror mods on repo software-definitions unit-test workspace"
```

Comment:
I might be misremembering how this works, but don't we also need to update the `ramble-completion.in`?

Reply:
I think we only need to update `ramble-completion.in` if we're expecting the ability to tab complete the actual tags or maintainers.

I think there's also a task to make sure this works properly.